### DISCUSSION TITLE
Change default loading spinner

### DISCUSSION THREAD

**Discussion by capouel50** [quasarframework/quasar#16887] [General - Components / Directives / etc] [answered]
I would like to change the default loading spinner with quasar 2.0.0 and vue 3.2.13.
The config in quasar-user-options.js doesn't work:
```js
import './styles/quasar.scss'
import lang from 'quasar/lang/fr.js'
import '@quasar/extras/material-icons/material-icons.css'
import { Notify, Loading, QSpinnerRings } from 'quasar'

// To be used on app.use(Quasar, { ... })
export default {
  plugins: {
    Loading,
    Notify
  },
  config: {
    loading: {
      spinner: QSpinnerRings,
      spinnerColor: 'red',
    },
  },
  lang: lang
}
```

**Comment by yusufkandemir** [comment]
Couldn't reproduce, it's working fine:
https://stackblitz.com/edit/quasar-framework-vite-plugin-ts-uxnddd?file=src/main.ts,src/App.vue

**Reply by capouel50** [reply]
Actually I would like to change the default spinner in a q-card. I have a loop that displays q-card dynamically to display raw material data. Each q-cards displays a spinner when loading data but I cannot modify the spinner.

**Reply by yusufkandemir** [reply]
QCard doesn't have a loading feature, you have some custom code and expect to get help without sharing it. You need to share your code to get help.

**Reply by capouel50** [reply]
`

            <div class="col-2 q-pa-sm" v-for="matiere in filteredMatieres" :key="matiere.id" `@click="redirectToLink(matiere.id)">

                <q-card bordered
                        class="card-maxi justify-center items-center text-center relative">

                  <div @click="redirectToLink(matiere.id)" class="card-content">
                    <q-img class="logo-card-mini"
                           src="../../assets/img/blanc.jpg"
                           :alt="matiere.designation"
                    >
                      <q-icon
                          :name="matiere.fournisseur.name === 'Cooper' ? 'copyright' : 'info'"
                          class="icon-background"
                      />
                      <div class="absolute-top hover-effect q-my-none text-cyan-1">
                        <div>{{ matiere.designation }}</div>
                        <div>{{ matiere.prix }}€ - {{ matiere.qté }}{{ matiere.unite }}</div>
                        <div>Ref fournisseur : {{ matiere.code_fournisseur }}</div>
                        <div>EAN : {{ matiere.cip }}</div>
                      </div>
                    </q-img>
                  </div>

                  <div v-if="matiere.cmr" class="absolute-bottom-left">
                    <font-awesome-icon fade icon="fa-solid fa-skull-crossbones" class="q-ml-xs q-mb-xs fa-2x" style="color: #e57373;"/>
                  </div>

                  <q-btn-group class="absolute-bottom-right q-pa-none q-ma-none">
                    <q-btn
                       class="q-pa-none hover-effect"
                       flat
                       color="cyan-4"
                       icon="add_box"
                       @click.stop="openDialog(matiere)"
                    />
                  </q-btn-group>

                </q-card>
              </div>`

**Reply by capouel50** [reply]
This is the loop that iclude q-card

**Reply by yusufkandemir** [reply, marked as answer]
You are using QImg which has a loading mechanism. You need to use the `#loading` slot, there are some examples in the docs: https://quasar.dev/vue-components/img#example--loading-state

**Reply by capouel50** [reply]
Yes i found the answer... thanks a lot
